niviz_rater/api.py

```diff
@@ -1,11 +1,12 @@
-'''
+"""
```

Comment:
I see you have a preference for " for blocks, we'll propagate this to other modules as another style-focused PR

Reply:
It's a side-effect from the editor - python generally uses """ for docstrings, not '''